packages/list-branches/action.yml

```diff
@@ -0,0 +1,32 @@
+name: 'List branches on repository'
+description: 'You can customize the message and the channel id.'
+inputs:
+  branche_regex:
+    description: 'The slack channel id'
+    required: true
+    default: '^master$|^dev$|release-.*'
+outputs:
+  branches:
+    description: "Json object who contains each branch"
+    value: ${{ steps.extract_branches.outputs.branches }}
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Install jq and jo
```

Comment:
I think `jq` is already installed on the runner

Reply:
Yes it is ! I remove it